test/doc/math/pedantic/hypot.cpp

```diff
@@ -8,12 +8,15 @@ using wide_ft = eve::wide<float, eve::fixed<4>>;
 
 int main()
 {
-  wide_ft pf = {-1.0f, eve::inf(eve::as(1.0f)), -3.0f, eve::valmax(eve::as<float>())/2};
+  wide_ft pf = {-1.0f, eve::inf(eve::as(1.0f)), -3.0f, eve::smallestposval(eve::as<float>())/2};
```

Comment:
Why changing the test instead of adding one ?

Reply:
it's a doc test. so I wanted to show the case that failed before. but if you want more outputs I can do that.